FakeXrmEasy.Tests.Shared/FakeContextTests/FetchXml/ConditionOperatorTests.cs

```diff
@@ -1616,19 +1715,22 @@ public void FetchXml_Operator_Next_Seven_Days_Execution()
             Assert.Equal("anniversary", query.Criteria.Conditions[0].AttributeName);
             Assert.Equal(ConditionOperator.Next7Days, query.Criteria.Conditions[0].Operator);
 
-            var date = DateTime.Now;
-            var ct1 = new Contact() { Id = Guid.NewGuid(), Anniversary = date.AddDays(3) }; //Should be returned
-            var ct2 = new Contact() { Id = Guid.NewGuid(), Anniversary = date.AddDays(7) }; //Shouldnt
+            var date = DateTime.UtcNow;
+            var ct1 = new Contact() { Id = Guid.NewGuid(), Anniversary = date.AddDays(3) }; // Should be returned
+            var ct2 = new Contact() { Id = Guid.NewGuid(), Anniversary = date.AddDays(7) }; // Should be returned
```

Comment:
I believe that "exactly 7 days away" is expected to be returned by "next 7 days".